_pages/support.md

```diff
@@ -0,0 +1,87 @@
+published: true
+site-map: true
+permalink: /support.html
+---
+
+# Your funding of pyOpenSci directly supports diversity in open source and open science  
+
+pyOpenSci builds a diverse community that supports the scientific Python software that scientists need for open science. Our  peer review program, packaging support, and training helps scientists build better, maintainable and trusted open source tools.  
+
+## Open source and open science are intertwined  
+
+Science is now data-driven. At pyOpenSci we value open science workflows because they allow scientists to build upon one another’s work and accelerate advancements, rather than spending time reinventing the wheel. Free and open software lowers open science barriers by allowing scientists to freely access the tools that they need to process their data.  
+
+## Diverse community is the heart of our work  
+
+Diversity in open source is even lower than it is in the broader tech community.  There is a lack of diversity across all segments in both tech and open source. But, it doesn’t have to be this way. pyOpenSci actively creates welcoming, supportive, safe, and inclusive spaces where all members feel represented, heard, and valued.  
+
+## Give today  
+
+pyOpenSci is currently seeking corporate and institutional sponsors. Your fiscal contribution will directly support the pyOpenSci mission by funding programs that:  
+- Improve the quality and usability of scientific Python software through community-driven open peer review  
+- Diversify and empower the  community through training and mentorship  
+- Build community capacity to contribute to open software  
+- Create supportive community for those developing scientific Python software that scientists need  
+
+To learn more about how you can support pyOpenSci, please email [admin@pyopensci.org](admin@pyopensci.org).  
+
+## Current funders and sponsors  
+
+PyOpenSci is currently sponsored by both the [Alfred P. Sloan Foundation](https://sloan.org/) and the [Chan Zuckerberg Initiative](https://chanzuckerberg.com/).  
+
+[SLOAN LOGO]
+
+
+<div class="feature__wrapper" markdown="1">
+   <div class="feature__item--left">
+      <div class="archive__item">
+         <div class="archive__item-teaser">
+            <img src="/images/sloan-square-logo.png" alt="">
+         </div>
+         <div class="archive__item-body">
+            <h2 class="archive__item-title">Alfred P. Sloan Foundation</h2>
+            <div class="archive__item-excerpt">
+               <p>The Alfred P. Sloan Foundation supports education and research related to science, technology, engineering, mathematics, and economics. Their mission is to “make the world a better place by advancing the frontiers of knowledge. We pledge to pursue that mission in ways that make the institutions of scientific endeavor more diverse, more equitable, and more inclusive to all.” You can learn more about the Sloan Foundation [on their About page](https://sloan.org/about).  
```

Comment:
Im going to fix this locally. the reason why the url is not rendering is that you have an html block with a <p> tag and markdown within it. so it either needs to be markdown or html but not a mix of both (exceptions for divs with markdown="1" supports both). 

but this specific layout is better suited to non html - a header yaml element for each of these blocks. so i'm going to redo this section and pull all of the text and image content up into the yaml and add an include :) the page will be cleaner that way!

Reply:
thaaaank you! I knew it was a mess, and now that I know about getting to the rendered page through artifacts any new pages I make will (hopefully 😅 ) need less finessing!

Reply:
oh no problem. also i forgot to tell you. IF you can build the site locally, you can view an orphan page but just typing the url. so here it would be something like:

localhost:4000/sponsors.html 

but yes the CI is here for us to make life easier so we don't always have to build. when i push changes i've made i'll comment on what i did and where just like i did for the volunteer page! the problem here is that this site is somewhat hacked together by me so it's not the most straight forward site to work with.